docs/api-reference/integration-guide.rst

```diff
@@ -96,12 +96,34 @@ Query PyPI’s `JSON
 API <https://warehouse.pypa.io/api-reference/json/>`__ to
 determine where to download files from.
 
-If you so choose
+Predictable URLs
 ----------------
 
-You can use our conveyor service to fetch this file, but this
-realistically exists primarily to support that historical predictable
-URLs still work.
+You can use our conveyor service to fetch this file, which exists for
+cases where using the API is impractical or impossible. This is for
+example the case for Linux package maintainers, as package build scripts
+or package metadata expect static URLs in some cases.
+
+URLs can be constructed as follows, with wheel file names following
+:pep:`491#file-name-convention`.
```

Comment:
Looks like the linter doesn't like this line, I'm not sure if we can use the `:pep:` directive with anchor tags like this.

Reply:
I answered this question 3 years ago :wink: https://github.com/pypi/warehouse/pull/5536#issuecomment-470542743

> PS: the travis failure is a bug in the validation: ``:pep:`number#anchor``` is valid syntax.

Reply:
Indeed you did!

Since this seems like a docutils bug (that hasn't been fixed in the last 3 years) we should find a way to either override this or just make this a regular link.

Reply:
We can make a PR to try removing the pins, but I don't see a recent release at https://pypi.org/project/doc8/ or https://pypi.org/project/docutils/, am I missing something?

Reply:
Found the issue: https://github.com/PyCQA/doc8/issues/124. I made a PR to unconstrain these in https://github.com/pypi/warehouse/pull/12711, will update that with the latest version when the release is made.

Reply:
we can leave the lower bound (>=1.1.0) to this PR, that version is not needed outside of it, right?

Reply:
I don't think it's strictly necessary, because we lock our requirements files, but it doesn't hurt.

Reply:
OK, I merged the main branch. It should work now!

Reply:
The version of `doc8` with your changes still hasn't been released.

Reply:
oh, whoops!